sns_ik_examples/src/ik_tests.cpp

```diff
@@ -209,6 +210,37 @@ void test(ros::NodeHandle& nh, double num_samples_pos, double num_samples_vel, s
   assert(chain.getNrOfJoints() == al.data.size());
 
   // SNS Position Tests
+  struct velocitySolverData {
+    sns_ik::VelocitySolveType type;
+    std::string               name;
+    double                   score;
+    double           scaling_score;
+    double                avg_time;
+  };
+  std::vector<velocitySolverData> vel_solver_data;
+  velocitySolverData sns = {sns_ik::SNS,"SNS",0.0,0.0};
```

Comment:
Oh, interesting. That's because I added `scaling_score` field afterward. I'll modify the code.